app/infrastructure/external/email/oauth_drivers.py

```
@@ -0,0 +1,316 @@
+            "prompt": "consent",
+            "include_granted_scopes": "true",
+        }
+
+    async def get_user_info(self, access_token: str) -> OAuthUserInfo:
+        async with httpx.AsyncClient() as client:
+            response = await client.get(
+                "https://gmail.googleapis.com/gmail/v1/users/me/profile",
+                headers={"Authorization": f"Bearer {access_token}"},
+            )
+            if response.status_code != 200:
+                raise ValueError(f"Failed to get user info: {response.text}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)